app/src/main/java/com/example/sharingang/database/firestore/FirestoreItemStore.kt

```diff
@@ -26,4 +29,14 @@ class FirestoreItemStore @Inject constructor(firestore: FirebaseFirestore) :
             }
         }
     }
+
+    override suspend fun getLastTimeUpdate(id: String): Date {
+        return firestore.collection(DatabaseFields.DBFIELD_ITEMS)
+            .document(id).get().await().getDate(DatabaseFields.DBFIELD_UPDATED_AT)!!
```

Comment:
What if the document was never update before?

Reply:
When the item is created, it is considered to be an update, therefore the "updatedAt" field will be non-null.